packages/build-tools/src/builders/ios.ts

```diff
@@ -106,7 +106,7 @@ async function buildAsync(ctx: BuildContext<Ios.Job>): Promise<void> {
 
     await ctx.runBuildPhase(BuildPhase.CONFIGURE_EXPO_UPDATES, async () => {
       await configureExpoUpdatesIfInstalledAsync(ctx, {
-        resolvedRuntimeVersion: resolvedExpoUpdatesRuntimeVersion,
+        resolvedRuntimeVersion: resolvedExpoUpdatesRuntimeVersion?.runtimeVersion ?? null,
```

Comment:
```suggestion
        resolvedRuntimeVersion: resolvedExpoUpdatesRuntimeVersion?.runtimeVersion ?? null,
        resolvedFingerprintSources: resolvedExpoUpdatesRuntimeVersion?.fingerprintSources ?? null,
```

Shouldn't this also have  `resolvedFingerprintSources` passed?

Reply:
Yes! Good spot. You can tell I tend to use the Android build for local testing 😳